### DISCUSSION TITLE
Oidc with Immich & iOs

### DISCUSSION THREAD

**Discussion by spammads** [authelia/authelia#4444] [Q&A] [answered]
In order for the companion app of Immich to work with Authelia as Identity Provider it is suggested to use `app.immich:/` as redirect-url in the configuration. But authelia only supports `https` or `http` as a scheme. Does anybody have a workaround for this?
Thanks & Keep up the good work.

Screenshot taken from https://immich.app/docs/usage/oauth
<img width="471" alt="Screenshot 2022-11-28 at 16 33 01" src="https://user-images.githubusercontent.com/63287042/204317573-19e1a39d-f476-43de-934a-065d249c04a4.png">

**Comment by james-d-elliott** [comment, marked as answer]
This is already fixed in [master](https://github.com/authelia/authelia/commit/c481ac86bbcf91cd551fb3c50c776d4bbb6872ed).
